src/redisjson.rs

```diff
@@ -437,23 +420,10 @@ pub mod type_methods {
         let json = match encver {
             0 => RedisJSON {
                 data: backward::json_rdb_load(rdb),
-                value_index: None, // TODO handle load from rdb
             },
             2 => {
                 let data = raw::load_string(rdb);
-                let schema = if raw::load_unsigned(rdb) > 0 {
-                    Some(ValueIndex {
-                        key: raw::load_string(rdb),
-                        index_name: raw::load_string(rdb),
```

Comment:
A bit far fetched, but is it possible that users extended the module and are writing additional module data to the db, and we should just consume (and ignore) our old schema data to avoid breaking user extension?

Reply:
Do you mean the redisearch 2.0 schema support that we just removed?